features/fixtures/ios/iOSTestApp/ViewController.swift

```diff
@@ -46,6 +51,43 @@ class ViewController: UIViewController {
         Scenario.clearPersistentData()
     }
 
+    func loadMazeRunnerAddress() -> String {
+
+        let bsAddress = "http://bs-local.com:9339"
+        
+        // Only iOS 12 and above will run on BitBat for now
```

Comment:
```suggestion
        // Only iOS 12 and above will run on BitBar for now
```

Reply:
I make this typo more frequently than I care to mention!  I've corrected it on my next branch (following on immediate) to avoid an extra CI run.

Reply:
💿🦇